source/php/Component/Hero/Hero.php

```diff
@@ -64,5 +64,28 @@ public function init()
         } else {
             $this->data['hasAnimation'] = false;
         }
+
+        /* Different hero types */
+        if($type) {
+            $this->data['contentSlotHasData'] = $this->slotHasData('content');
+            if(is_string($type)) {
+                $this->data['type'] = 'Hero.views.' . $type;
+                $this->data['classList'][] = $this->getBaseClass() . '--' . $type;
+            } else {
+                $this->data['type'] = 'Hero.views.' . $type['type'];
+                $this->data['classList'][] = $this->getBaseClass() . '--' . $type['type'];
+                $this->data['background'] = $type['background'] ? $type['background'] : ($type['image'] ? 'url(' .$type['image'] . ')' : '');
```

Comment:
Send to attributeList with the key $style and it will automatically be santitized if possible.

Reply:
The reason behind this is because the background is not applied on the parent element but a level below to make it work with the blur effect of the background image.